### PR TITLE
Week 48

### PR DESCRIPTION
**Pull request by lbarcziova**

### CODE REVIEW

File: content/posts/weekly/December-2022.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- `packit propose-downstream` now uploads all remote sources (those specified as URLs) and the
+  source specified by `spec_source_id` (whether remote or not) to lookaside. Previously, only Source0
+  was uploaded. Source0 is no longer treated specially, but as `spec_source_id` is `Source0` by default,
+  Source0 is still being uploaded by default unless `spec_source_id` is overriden.
+  ([packit#1778](https://github.com/packit/packit/pull/1778))
+- [A VM image build](/docs/configuration##vm_image_build) can be triggered inside a PR via a comment
+  command `/packit vm-image-build` (the job needs to be defined in the configuration).
+  This feature is experimental and is still being tested.
+  ([packit-service#1761](https://github.com/packit/packit-service/pull/1761))
```
[the remark this answers is not in the document]
Comment (majamassarini) [reply content/posts/weekly/December-2022.md:17]: I was just wondering if we should define the job for every packit package (ogr, specfile, packit) and create another daily validation test to stress this before announcing it?

**Review by nforro — APPROVED** [review]

File: content/docs/Configuration/_index.md [hunk 2]
```diff
@@ -830,13 +830,9 @@ jobs:
 
 ##### vm_image_build
 
-**PACKIT GITHUB APP IS NOT PROCESSING THIS JOB IN ANY WAY.**
-
-Before we implement this in the app, please refer to the [CLI guide]({{< ref "in-image-builder.md" >}}).
-
```
Comment (TomasTomecek) [inline content/docs/Configuration/_index.md:836]: well done!

File: content/posts/weekly/December-2022.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lbarcziova) [reply content/posts/weekly/December-2022.md:17]: agree, it's better to test it properly before announcing (and we can also update the docs if needed)

[the remark this answers is not in the document]
Comment (TomasTomecek) [reply content/posts/weekly/December-2022.md:17]: I would keep this here and do a proper (big) announcement after :)

Agreed with Maja to do more testing before announcing the big one.

**Review by TomasTomecek — APPROVED** [review]

File: content/posts/weekly/December-2022.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nforro) [reply content/posts/weekly/December-2022.md:17]: > I was just wondering if we should define the job for every packit package (ogr, specfile, packit) and create another daily validation test to stress this before announcing it?

That's a good idea, but I think we can announce it anyway, it's clearly stated that it's an experimental feature.

**Review by lachmanfrantisek — APPROVED** [review]

File: content/posts/weekly/December-2022.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lbarcziova) [reply content/posts/weekly/December-2022.md]: ok, I will re-add the image builder related changes :D thanks for the feedback!

**Pull request merged**